lib/screens/custom_drawer.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:e_wallet/providers/auth_provider.dart';
+import 'package:e_wallet/screens/contact_list_screen.dart';
+import 'package:e_wallet/screens/transaction_history_screen..dart';
+import 'package:flutter/material.dart';
+
+class CustomDrawer extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      child: ListView(
+        padding: EdgeInsets.zero,
+        children: <Widget>[
+          DrawerHeader(
+            child: Text(
+              'E Wallet',
+              style: TextStyle(
+                  fontSize: 30,
+                  color: Theme.of(context).primaryTextTheme.bodyText1.color),
+            ),
+            decoration: BoxDecoration(
+              color: Colors.blue,
+            ),
+          ),
+          ListTile(
+            title: Text('Transaction History'),
+            leading: Icon(Icons.assignment_turned_in),
+            onTap: () {
+              Navigator.pop(context);
+              Navigator.pushNamed(context, TransactionHistory().routeName);
```

Comment:
What happens when you remove `Navigator.pop(context);`?

Reply:
context on that part is the Navigator. It's used to close the navigator when opening a new activity. But If you remove it, the navigator will stay open when you press the back button.